pkgs/by-name/no/nofi/package.nix

```
@@ -0,0 +1,28 @@
+{ lib, rustPlatform, fetchFromGitHub, dbus, pkg-config}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nofi";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    owner = "ellsclytn";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hQYIcyNCxb8qVpseNsmjyPxlwbMxDpXeZ+H1vpv62rQ=";
```

```suggestion
    hash = "sha256-hQYIcyNCxb8qVpseNsmjyPxlwbMxDpXeZ+H1vpv62rQ=";
```
The sha256 part is redundant